### DISCUSSION TITLE
Wikipedia article?

### DISCUSSION THREAD

**Discussion by tkoenig1** [hneemann/Digital#793] [General]
To make Digital easier to find, would it make sense for it to have a Wikipedia article?

It could then be re-added to [its article on Free electronics circuit simulators](https://en.wikipedia.org/wiki/List_of_free_electronics_circuit_simulators) (from which it was removed due to not having one of its own).

**Comment by hneemann** [comment]
I started a Wikipedia article some time ago, but even while it was still in draft status, a moderator contacted me to tell me that it wasn't worth finishing because it couldn't be published due to a lack of suitable supporting evidence. And to be honest, I didn't feel like arguing about it.
But if you are up for it and have the time, you can try again! :smile:
